spec/basic/processedconfig_spec.js

```diff
@@ -0,0 +1,11 @@
+var util = require('util');
+
+describe('configuration elements', function() {
+  it('should have access to the processed config block', function() {
+    browser.getProcessedConfig().then(function(config) {
```

Comment:
This will actually fail in our CI where the browser is not always Chrome - I'll modify slightly.